fastlane/Fastfile

```diff
@@ -83,7 +83,9 @@ platform :ios do
     sync_code_signing(
       type: "appstore",
       readonly: true,
-      app_identifier: ["io.sentry.sample.iOS-Swift",  "io.sentry.sample.iOS-Swift.Clip"]
+      app_identifier: ["io.sentry.sample.iOS-Swift",  "io.sentry.sample.iOS-Swift.Clip"],
+      git_branch: "master",
+      clone_branch_directly: true
```

Comment:
`l`: Please comment on why we have to use this. Basically it fixes the rare fatal: a branch named 'master' already exists in GH actions.

Reply:
No, I think that's not required. We can always check what the flag does. The change fixes a problem with CI that's not obvious when you look at the code. I'm thinking of the future me that sees that in 1 year, has no clue why it's there, removes it, and then wonders why CI starts to fail again.

Reply:
Added comments